src/frontend/src/instance.rs

```diff
@@ -527,8 +530,8 @@ pub fn check_permission(
         },
         Statement::Copy(sql::statements::copy::Copy::CopyDatabase(copy_database)) => {
             match copy_database {
-                CopyDatabase::To(stmt) => validate_param(&stmt.database_name, query_ctx)?,
-                CopyDatabase::From(stmt) => validate_param(&stmt.database_name, query_ctx)?,
+                CopyDatabase::To(stmt) => validate_database(&stmt.database_name, query_ctx)?,
+                CopyDatabase::From(stmt) => validate_database(&stmt.database_name, query_ctx)?,
```

Comment:
Am I correct?  @v0y4g3r

Reply:
> Am I correct? @v0y4g3r

The original behavior is not correct since it treats database name as table idents. This change fixes it.